ring-jetty-adapter/test/ring/adapter/test/jetty.clj

```diff
@@ -290,6 +290,14 @@
       (is (= 1000 (. thread-pool getIdleTimeout)))
       (.stop server)))
 
+  (testing "providing custom thread-pool"
+    (let [pool   (QueuedThreadPool.)
+          server (run-jetty hello-world {:port test-port
+                                         :join? false
+                                         :thread-pool pool})]
+      (is (= pool (. server getThreadPool)))
```

Comment:
Instead of `=`, use `identical?`

Reply:
And instead of `(. server getThreadPool)`, use `(.getThreadPool server)`

Reply:
Changes made.